plugin-server/src/main/ingestion-queues/scheduled-tasks-consumer.ts

```diff
@@ -63,26 +63,13 @@ export const startScheduledTasksConsumer = async ({
             })
             const startTime = performance.now()
 
-            // Make sure tasks can't run forever, according to `taskTimeouts`.
-            const abortController = new AbortController()
-            const timeout = setTimeout(() => {
-                abortController.abort()
-                status.warn('⚠️', 'scheduled_task_timed_out', {
```

Comment:
We do see this log happening on prod [[loki query]](http://grafana-prod-us/explore?orgId=1&left=%7B%22datasource%22:%22P8E80F9AEF21F6940%22,%22queries%22:%5B%7B%22refId%22:%22A%22,%22expr%22:%22%7Bcontainer%3D%5C%22posthog-plugins-scheduler%5C%22%7D%20%7C%3D%20%60scheduled_task_timed_out%60%22,%22queryType%22:%22range%22,%22editorMode%22:%22builder%22%7D%5D,%22range%22:%7B%22from%22:%22now-24h%22,%22to%22:%22now%22%7D%7D). Will another timeout trigger instead?

Reply:
Yeah I think it's worth taking the gamble here. The only thing I believe that this is doing is causing the IPC to timeout rather than the actual task run, so we'll lose a bit of backpressure, but we'll see.

Reply:
Works for me!